mesh/meshdb.go

```diff
@@ -77,11 +77,18 @@ func (m *MeshDB) GetBlock(id types.BlockID) (*types.Block, error) {
 		return nil, err
 	}
 
-	transactions, err := m.getTransactions(blk.TxIds)
-	if err != nil {
+	txs, missing := m.GetTransactions(blk.TxIds)
+
+	if missing != nil {
 		m.Error("could not retrieve block %v transactions from database ")
 		return nil, err
 	}
+
+	transactions := []*types.SerializableTransaction{}
```

Comment:
see comment about GetTransactions. maybe just return a SerializableTransaction slice instead of doing the copy below